prometheus-connector/src/main/java/org/apache/flink/connector/prometheus/sink/prometheus/Remote.java

```diff
@@ -0,0 +1,6661 @@
+/*
```

Comment:
Should we consider adding API snapshot tests for this class?

Reply:
Same for `Types.java` and `GoGoProtos.java`

Reply:
I think we discussed this.
Generating the Java classes from proto cannot be done with a pure Java solution included in the project. 
It requires ProtoBuf binaries. (e.g. see https://github.com/javiroman/jremotewrite/blob/e1b5d9865d42254b32f2da6ddfd6956569577601/pom.xml#L210)

 I am adding the *.proto file for reference and a section to the README to explain